Locales/Russian_ru.json

```diff
@@ -13,1301 +13,1316 @@
 // REMEMBER: You do **ONLY** need to change the right-side text. If translations suddenly stop working, check your `cl_locale` variable and make sure the JSON file structure was not damaged.
 
 {
+
 //// SCOREBOARD ////
 //ScoreboardCommon.as
-	"Match time: {TIME}": "Время матча: {TIME}",
-	"h ": "ч ",
-	"m ": "м ",
-	"s ": "с ",
+  "Match time: {TIME}": "Время матча: {TIME}",
```

Comment:
Tabs should not have been modified to spaces in this PR, I will fix that.

Reply:
Ok, sorry